release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -462,6 +471,11 @@ In the table below, features are marked with the following statuses:
 |GA
 |GA
 
+|`SharedSecret` objects for {op-system-base} entitlements
```

Comment:
@gabemontero Let's find a nice short name for the BUILD-347 feature. Here's a first pass.

Reply:
- "SharedSecret objects for RHEL entitlements" in  https://deploy-preview-40899--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes.html#ocp-4-10-technology-preview

Reply:
Given the first line of https://docs.openshift.com/container-platform/4.9/cicd/builds/running-entitled-builds.html let's build off of that, and say

"Entitled Builds with 'SharedSecrets'"

Reply:
Applied, thanks.